src/main/java/it/gov/pagopa/payhub/activities/connector/organization/client/BrokerClient.java

```diff
@@ -0,0 +1,26 @@
+package it.gov.pagopa.payhub.activities.connector.organization.client;
+
+import it.gov.pagopa.payhub.activities.connector.organization.config.OrganizationApisHolder;
+import it.gov.pagopa.pu.organization.dto.generated.PagedModelBroker;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Lazy
+@Service
+public class BrokerClient {
+	private static final int DEFAULT_PAGE_NUMBER = 0;
+	private static final int DEFAULT_PAGE_SIZE = 20;
```

Comment:
This probably is the default for the service, but we are not interested on the default, we want  all Brokers, lets configure to the maximum available (or a number reasonably high, eg: 2000)

Don't use a constant, neither for pageNumber... we probably have no usage for this values on this class